reflex/components/radix/primitives/slider.py

```diff
@@ -0,0 +1,170 @@
+"""Radix slider components."""
+
+from typing import Any, Dict, Literal
+
+from reflex.components.component import Component
+from reflex.components.radix.primitives.base import RadixPrimitiveComponent
+from reflex.style import Style
+from reflex.vars import Var
+
+LiteralSliderOrientation = Literal["horizontal", "vertical"]
+LiteralSliderDir = Literal["ltr", "rtl"]
+
+
+class SliderComponent(RadixPrimitiveComponent):
+    """Base class for all @radix-ui/react-slider components."""
+
+    library = "@radix-ui/react-slider@^1.1.2"
+
+
+class SliderRoot(SliderComponent):
+    """The Slider component comtaining all slider parts."""
+
+    tag = "Root"
+    alias = "RadixSliderRoot"
+
+    default_value: Var[list[int]]
+
+    value: Var[int]
+
+    name: Var[str]
+
+    disabled: Var[bool]
+
+    orientation: Var[LiteralSliderOrientation]
+
+    dir: Var[LiteralSliderDir]
+
+    inverted: Var[bool]
+
+    min: Var[int]
```

Comment:
in the chakra components we have added underscore suffix to names that shadow python builtins, will we do the same here?

personally i think _NOT_ changing the names is better, as long as they will never be used as unqualified names, then it's better to have consistency with the upstream component, and the shadowing won't actually matter (it also looks better than having a bunch of underscore suffixes)

Reply:
We need to discuss with the team to decide on this yeah.